control/controlclient/direct.go

```diff
@@ -467,9 +467,9 @@ func (c *Direct) doLogin(ctx context.Context, opt loginOpt) (mustRegen bool, new
 	//	- user is disabled
 
 	if resp.AuthURL != "" {
-		c.logf("AuthURL is %v", resp.AuthURL)
+		c.logf("[v1] AuthURL is %v", resp.AuthURL)
```

Comment:
This is the log message which prints https://login.tailscale.com/a/0123456789 ? We do have people who have used this, to see the login URL in cloud instance logs for example so they can authenticate the system even without having SSH access to see the output of tailscale up.

Reply:
Reverted this line.